src/lein_monolith/task/fingerprint.clj

```diff
@@ -325,14 +337,16 @@
    ::sources
    ::deps
    ::java-version
-   ::upstream])
+   ::upstream
+   ::jar-exclusions])
```

Comment:
Minor, but inverting these might fit better. So far the list is ordered so that all of the local configuration changes are "more important" explanations of a fingerprint changing than the upstream dependency fingerprints.
```suggestion
   ::jar-exclusions
   ::upstream])
```